Makefile

```diff
@@ -53,9 +54,9 @@ watch: pdf
 # Cleans all intermediate files except for the produced pdf files.
 clean:
 	$(LATEXMK) -C
-	# latexmk does not clean up the aux-files produced by the chapters.
-	# They are stand-alone compilation units with a dedicated aux file.
-	# This comes because we include them with "\include" instead of "\input".
+	$(QUIET) latexmk does not clean up the aux-files produced by the chapters.
```

Comment:
`#` is missing here, if I am not mistaken?

Reply:
good catch!